### PR TITLE
Complete documentation for BroydenWrapper.

### LINKED ISSUE #10: Add xml documentation to BroydenWrapper
**Context** 
The best practices requires xlm documentation for publicly exposed elements.

**Details**
QuantSA/QuantSA.Core/RootFinding/BroydenWrapper.cs

### PR DESCRIPTION
**Pull request by rwx-oberholzer-marcelle**
# ✅ Pull Request Checklist

Before submitting your pull request, please ensure the following are complete:

## 🔍 Self Review
- [x] Confirmed that the destination branch is correct.
- [x] I have reviewed my own code carefully.
- [x] I have removed any debugging code or print statements.
- [x] Code changes are scoped to the purpose of this PR (no unrelated changes).
- [x] Comments are clear, and the code is self-explanatory where possible.
- [x] Documentation (if applicable) is updated.

## 📄 Task
- [x] The task/issue title is clear and descriptive.
- [x] The description explains **why** (context) and **what** (details) clearly.
- [x] Related issues are referenced (e.g., "Fixes #123").

## ✅ Code Quality
- [x] There are no new warnings or errors in the build output.
- [x] Code follows the project’s coding standards and formatting rules.

## 🧪 Testing
- [x] All unit and integration tests pass locally.
- [x] New functionality is covered by tests, or is explicitly marked as not testable.
- [x] Edge cases and error handling have been considered.

## ✅ CI
- [x] CI checks are passing (e.g., build, tests, linting).

## 📄 PR Quality
- [x] The PR title is clear and descriptive.
- [x] The PR description explains **what** and **why** clearly.
- [x] Related issues are referenced (e.g., "Fixes #123").

## 👥 Collaboration
- [x] I have assigned reviewers and/or requested feedback.
- [x] I’ve addressed or responded to all prior review comments (if applicable).

### CODE REVIEW

File: QuantSA/QuantSA.Core/RootFinding/BroydenWrapper.cs [hunk 1]
```diff
@@ -11,6 +11,13 @@ namespace QuantSA.Core.RootFinding
     /// </summary>
     public class BroydenWrapper : IVectorRootFinder
     {
+
```
Comment (rwx-jacquier-robert) [inline QuantSA/QuantSA.Core/RootFinding/BroydenWrapper.cs]: Spurious newline.

**Review by rwx-jacquier-robert — CHANGES_REQUESTED** [review]

**Review by rwx-jacquier-robert — APPROVED** [review]
